### PR TITLE
html-basics-practice

### PR DESCRIPTION
**Pull request by Abbas4tech**
html practice

### CODE REVIEW

File: tasks-by-users/AbbasShaikh0304/p1/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+  <html>
+      <head>
+          <meta charset="utf-8">
+          <meta name="keywords" content="Automobile, parts, garage">
+          <meta name="Description" content="Welcome to our garage to get parts at most affordable prizes">
+          <title>Welcome to Online Book Store</title>
+      </head>
+      <body>
+          <header><img src="/img/Logo.png" width="100%" height="150px" alt="oops!"></header>
```
[the remark this answers is not in the document]
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/p1/index.html:10]: I'll correct it😅

**Review by Naufil-Shemle — APPROVED** [review]

File: tasks-by-users/AbbasShaikh0304/p1/About-Us.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (fayez-baig) [inline tasks-by-users/AbbasShaikh0304/p1/About-Us.html:6]: hey

File: tasks-by-users/AbbasShaikh0304/p1/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+              <button>Join Now!</button>
+              <ul>
+                  <li>
+                  Follow Us
+                <ul>
+                    <li><a href="">FaceBook</a></li>
+                    <li><a href="">Twitter</a></li>
+                    <li><a href="">YouTube</a></li>
+                    <li><a href="">Instagram</a></li>
+                </ul>
```
Comment (coder-mezab) [inline tasks-by-users/AbbasShaikh0304/p1/index.html:34]: why `ul` in `li`

File: tasks-by-users/AbbasShaikh0304/p1/About-Us.html [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+</head>
+<body>
+    <p>We sell the books at most affordable ranges.<br> Online Book store is an online web application where the customer can purchase books online. Through a web browser the customers can search for a book by its title or author, later can add to the shopping cart and finally purchase using credit card.<br>
+        <a href="/index.html">Back</a>
```
Comment (faran4engg) [inline tasks-by-users/AbbasShaikh0304/p1/About-Us.html:10]: indentation ? add prettier in VSCode.. and do necessary setting

File: tasks-by-users/AbbasShaikh0304/p1/index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+  <html>
+      <head>
+          <meta charset="utf-8">
+          <meta name="keywords" content="Automobile, parts, garage">
+          <meta name="Description" content="Welcome to our garage to get parts at most affordable prizes">
+          <title>Welcome to Online Book Store</title>
+      </head>
+      <body>
+          <header><img src="/img/Logo.png" width="100%" height="150px" alt="oops!"></header>
+          <h1>
```
Comment (faran4engg) [inline tasks-by-users/AbbasShaikh0304/p1/index.html:11]: why is this resolved? You should not resolve until you fixed it @AbbasShaikh0304

File: tasks-by-users/AbbasShaikh0304/p1/index.html (hunk 3 above)
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/p1/index.html:34]: It is list inside an another list. Check full code

File: tasks-by-users/AbbasShaikh0304/p1/About-Us.html (hunk 4 above)
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/p1/About-Us.html:10]: Gotcha 👍

File: tasks-by-users/AbbasShaikh0304/p1/index.html [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+            Online Book Store     
+          </h1>
+          <p>Welcome to <strong>online Book Store</strong>.We sell the Books of varies genre at most affordable prize. You can also download the digital copy of your Books.</p>
+          <h2>Genre</h2>
+          <hr>
+              <ul>
+                          <li><img src="/img/Comic.jpg" width="100px" height="100px"><a href="">Comic</a></li>
+                          <li><img src="/img/Biography.jpg" width="100px" height="100px"><a href="">Biography</a></li>
```
[the remark this answers is not in the document]
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/p1/index.html:19]: Just for example, creating it as link. I put empty href

File: tasks-by-users/AbbasShaikh0304/Readme.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+# Welcome
+
+###### These tasks are done by me only
```
[the remark this answers is not in the document]
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/Readme.md:3]: Yes @faran4engg

File: tasks-by-users/AbbasShaikh0304/p1/index.html [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+      <body>
+          <header><img src="/img/Logo.png" width="100%" height="150px" alt="oops!"></header>
+          <h1>
+            Online Book Store     
+          </h1>
+          <p>Welcome to <strong>online Book Store</strong>.We sell the Books of varies genre at most affordable prize. You can also download the digital copy of your Books.</p>
+          <h2>Genre</h2>
+          <hr>
+              <ul>
+                          <li><img src="/img/Comic.jpg" width="100px" height="100px"><a href="">Comic</a></li>
```
[the remark this answers is not in the document]
Comment (AbbasShaikh0304) [reply tasks-by-users/AbbasShaikh0304/p1/index.html:18]: Used prettier @fayez-baig

File: tasks-by-users/AbbasShaikh0304/p1/index.html (hunk 6 above)
[the remark this answers is not in the document]
Comment (faran4engg) [reply tasks-by-users/AbbasShaikh0304/p1/index.html:19]: Np. It's ok as it's your first task. Just commenting for you for the upcoming projects

**Review by faran4engg — APPROVED** [review]

**Comment by fayez-baig** [comment]
Please add @NextGenDevOrg/all-reviewers  in the reviewers tab @AbbasShaikh0304

**Review by fayez-baig — APPROVED** [review]
Ok

**Review by faran4engg — APPROVED** [review]

**Pull request merged**